### PR TITLE
Feature/define module provider

### PR DESCRIPTION
**Pull request by DarkoKukovec**

### CODE REVIEW

File: src/Provider.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export const Provider: React.FC<IProviderProps> = ({ children, parentScope, providers, scope, test }) => {
+  const contextScope = React.useContext(InjectionContext);
+  const parentScopeId = parentScope || (test ? undefined : contextScope);
+  const moduleScopeId = scope || (test ? undefined : new ScopeToken('CONTEXT_SCOPE'));
+
+  const moduleScope = test
+    ? GenerateTestBed(providers, moduleScopeId, parentScopeId)
+    : DefineModule(providers, moduleScopeId, parentScopeId);
+
+  return <InjectionContext.Provider value={moduleScope}>{React.Children.only(children)}</InjectionContext.Provider>;
```
Comment (isBatak) [inline src/Provider.tsx]: Why do we allow only one child?

Comment (isBatak) [reply src/Provider.tsx]: We are not China xD

Comment (DarkoKukovec) [reply src/Provider.tsx]: :D Good question... I'm not exactly sure if we need it...

**Comment by DarkoKukovec** [comment]
Now we can :D

**Pull request merged**